tests/yamitranscode.cpp

```diff
@@ -48,9 +48,9 @@ static void print_help(const char* app)
     printf("   --intraperiod <Intra frame period(default 30)> optional\n");
     printf("   --refnum <number of referece frames(default 1)> optional\n");
     printf("   --idrinterval <AVC/HEVC IDR frame interval(default 0)> optional\n");
-    printf("   --enablecabac <AVC is to use CABAC or not in Main Profile (default true)> optional\n");
-    printf("   --enabledct8x8 <AVC is to use DCT8x8 transform or not in High Profile (default false)> optional\n");
-    printf("   --enabledeblockfilter <AVC is to use Deblock filter or not (default true)> optional\n");
+    printf("   --disablecabac <AVC Main Profile is to use CABAC or not(default false)> optional\n");
+    printf("   --enabledct8x8 <AVC High Profile is to use DCT8x8 transform or not(default false)> optional\n");
```

Comment:
Better change to --enable-dct8x8
and "AVC is to use DCT8x8 or not, for High profile, default is disabled"